addons/website_blog/static/src/js/website_blog.inline.discussion.js

```diff
@@ -36,12 +36,13 @@ var InlineDiscussion = core.Class.extend({
         self.discussions_handler(self.settings.content);
 
         // Hide the discussion.
-        $('html').click(function(event) {
-            if($(event.target).parents('#discussions_wrapper, .main-discussion-link-wrp').length === 0) {
+        $('html').click(function (event) {
+            if ($(event.target).parents('#discussions_wrapper, .main-discussion-link-wrp').length === 0) {
```

Comment:
Please remove the typo change from the diff (don't add/remove spaces if you don't change the line)